algorithm_catalog/biopar.json

```diff
@@ -0,0 +1,122 @@
+        "cost_estimate": 4,
+        "cost_unit": "platform credits per km²",
+        "keywords": [
+            "agriculture",
+            "crops"
+        ],
+        "language": {
+            "code": "en-US",
+            "name": "English (United States)"
+        },
+        "languages": [
+            {
+                "code": "en-US",
+                "name": "English (United States)"
+            }
+        ],
+        "contacts": [
+            {
+                "name": "Jeroen Dries",
+                "position": "Researcher",
+                "organization": "VITO",
+                "links": [
+                    {
+                        "href": "https://www.vito.be/",
+                        "rel": "about",
+                        "type": "text/html"
+                    },
+                    {
+                        "href": "https://github.com/jdries",
+                        "rel": "about",
+                        "type": "text/html"
+                    }
+                ],
+                "contactInstructions": "Contact via VITO",
+                "roles": [
+                    "principal investigator"
+                ]
+            },
+            {
+                "name": "VITO",
+                "links": [
+                    {
+                        "href": "https://www.vito.be/",
+                        "rel": "about",
+                        "type": "text/html"
+                    }
+                ],
+                "contactInstructions": "SEE WEBSITE",
+                "roles": [
+                    "processor"
+                ]
+            }
+        ],
+        "themes": [
+            {
+                "concepts": [
+                    {
+                        "id": "Leaf Area Index (LAI)"
+                    },
+                    {
+                        "id": "Fraction of Absorbed Photosynthetic Active Radiation (fAPAR)"
+                    },
+                    {
+                        "id": "Fraction of Vegetation Coverage (fCOVER)"
+                    },
+                    {
+                        "id": "Sentinel-2 MSI"
+                    },
+                    {
+                        "id": "Canopy Water Content (CWC)"
+                    },
+                    {
+                        "id": "Canopy Chlorophyll Content (CCC)"
+                    }
+                ],
+                "scheme": "https://gcmd.earthdata.nasa.gov/kms/concepts/concept_scheme/sciencekeywords"
+            }
+        ],
+        "formats": [
+            "GeoTiff"
+        ],
+        "license": "other"
+    },
+    "linkTemplates": [],
+    "links": [
+        {
+            "rel": "openeo-process",
+            "type": "application/json",
+            "title": "openEO Process Definition",
+            "href": "https://raw.githubusercontent.com/ESA-APEx/apex_algorithms/refs/heads/biopar_vito/openeo_udp/biopar/biopar.json"
```

Comment:
This URL will break once the feature branch biopar_vito is removed

change it to https://raw.githubusercontent.com/ESA-APEx/apex_algorithms/3b5a011a90f4a3050ff8fdf69ca5bc2fd1535881/openeo_udp/biopar/biopar.json